src/python/WMCore/Services/RucioConMon/RucioConMon.py

```diff
@@ -71,9 +84,15 @@ def _getResultZipped(self, uri, callname="", clearCache=True, args=None):
         This method is retrieving a zipped file from the uri privided, instead
         of the normal json
         :param uri: The endpoint uri
-        :return:    A dictionary
+        :param callname: alias for caller function
+        :param clearCache: parameter to control the cache behavior
+        :param args: additional parameters to HTTP request call
+        :return:    a list of LFNs
         """
-        raise NotImplementedError
+        data = self._getResult(uri, callname, clearCache, args, binary=True)
+        # convert bytes which we received upstream to string
+        data = data.decode("utf-8")
```

Comment:
I'd suggest to replace this by the utilitarian function `decodeBytesToUnicode`, under:
https://github.com/dmwm/WMCore/blob/master/src/python/Utils/Utilities.py#L198

Reply:
done